docker-hub/api/deprecated.md

```diff
@@ -0,0 +1,29 @@
+---
+description: Docker Hub API v1 (deprecated)
+keywords: kitematic, deprecated
+title: Docker Hub API v1 (deprecated)
+---
+
+> **Deprecated**
+>
+> Docker Hub API v1 has been deprecated. Please use Docker Hub API v1 instead.
```

Comment:
@silvin-lubecki Should this be v2 instead?

Reply:
Of course... 😅

Reply:
Fixed https://github.com/docker/docker.github.io/pull/15461/commits/0f400bf78ba6571c338fa7fd89111a55d50861ad